packages/database/src/migrations/20210128102520-PalauFacilityOpeningHoursDashboard-modifies-data.js

```diff
@@ -68,12 +68,42 @@ const dashboardReport = {
   },
 };
 
+const elementList = [
+  'FF20',
+  'FF21',
+  'FF22',
+  'FF23',
+  'BCD39D',
+  'FF24',
+  'FF25',
+  'BCD39E',
+  'FF26',
+  'FF27',
+  'BCD39F',
+  'FF28',
+  'FF29',
+  'BCD39G',
+  'FF30',
+  'FF31',
+  'BCD39H',
+  'FF32',
+  'FF33',
+];
+const changeServiceType = async (db, serviceType) => {
+  await db.runSql(`
+    UPDATE data_source 
+    SET service_type = '${serviceType}'
+    WHERE code in (${arrayToDbString(elementList)})
+  `);
+};
 exports.up = async function (db) {
+  await changeServiceType(db, 'tupaia');
```

Comment:
This is quite a separate concern to the other aspects of this migration - I'd split it out (that way the feature branch also won't need relaunching)

Reply:
Thanks @edmofro for a doulbe review. Does it mean if there is a new migration file, it will automatically run it?

Reply:
That's right :-)